### DISCUSSION TITLE
理智讨论近期皮套节奏以及应对态度

### DISCUSSION THREAD

**Discussion by cyq20091029** [share121/inter-knot#2744] [其他]
<img width="346" alt="截屏2024-08-17 21 47 52" src="https://github.com/user-attachments/assets/5e92175f-d4a6-42f2-895d-9e9bb36993e5">
欢迎各位绳匠讨论

**Comment by March-777th** [comment]
评价为闹麻了，多看一眼都是对智商的侮辱<img src="https://bbs-static.miyoushe.com/static/2024/07/31/73346f2b98d45b62da9139d2774fa856_8641114165088315754.png" alt="三月七 告退" width="75" >

**Comment by shouyeren7** [comment]
换汤不换药，污名化模板

**Comment by OttoApocalypse01** [comment]
只能说它们已经没活了，只能这样撒泼打滚了。多看一眼我都觉得魔阴身要犯了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by wodongx123** [comment]
没有看到什么有营养的论据，观点重要论证过程也很重要不是吗？
最核心的论据就是这个像那个像，也没有很像啊？
是不是接下来还要切香肠，鼻子像脸像嘴巴像反正有一个像就是像

**Comment by Nie7bai** [comment]
是真没活力，不如他们前辈的赛博龙脉有石粒

**Comment by wu364** [comment]
不如赛博龙脉一根<img src="https://bbs-static.miyoushe.com/static/2023/10/10/6fe05fae170b754c2c1afe3f1f6c847c_2945463426120653954.png" alt="卡琳_wink" width="75" >

**Reply by nabeilishi** [reply]
河南拔智齿

**Comment by ssOneNight** [comment]
我认为没有任何营养，捕风捉影+大肆扩散洗脑包，现在就连所谓的证据是否为真都没意义了，已经扩散到刻板影响恶心人的程度了

**Comment by HKLov3** [comment]
纯就是黑子们实在没活了，以前就有，纯就是“某条还不能算人的玩意”在断章取义混淆视听吃黑流量

**Comment by ShanFengg** [comment]
一边觉得原神玩家都是现充，一边觉得皮套节奏能让玩家破防，评价为闹麻了

**Comment by Kujojolen** [comment]
我直接不放心上，节奏起它的，我玩我的游戏，根本不影响

**Comment by qtsa66** [comment]
按照他们的逻辑我可以得出（你看你和你爹很像，然后呢你和你爹一个姓，再然后是他“创造”了你，所以你是你爹的皮套）

**Comment by GGGGG666654** [comment]
还是有点门槛才不会阴兵过境啊 感觉这才是正常的讨论氛围qwq

**Comment by TetsuyaTetsuya** [comment]
最若只的一集，只能说谁信谁就是真若只

**Comment by demolute** [comment]
很正常，佐助还是我的皮套呢，当初岸本为了写佐助一直都在跟踪我

**Comment by AcheronBlackswanForever** [comment]
评价就是真理医生的一句话：0分，给我滚。<img src="https://bbs-static.miyoushe.com/static/2023/12/19/e0b413ade715eb75aba2731c839bb724_6900592663590997950.png" alt="真理医生 计算中" width="75" >
还有什么评价就是尾巴大爷的一句话：给老子滚！<img src="https://bbs-static.miyoushe.com/static/2023/11/07/722fc5f2e631e5c02184afdc15c2209f_7114079448845986183.png" alt="藿藿 抓狂" width="75" >

**Comment by fuqiam** [comment]
看看克苟的切片评价一针见血

**Comment by SPIKE-spike** [comment]
我现在就想知道这个到底是个什么事情 B站上对于这件事基本上都是情绪输出 我瓜都还没吃明白呢

**Reply by cjzhou0604** [reply]
你说的对，那些纯粹情绪输出的文字或者视频我真的不想再看到了，但是又说不得，谁知道他们会喷些什么东西

**Comment by AnkaAlioth** [comment]
不知道发生了什么的别去看，纯纯的恶心人罢了<img src="https://img-static.mihoyo.com/communityweb/upload/366bd6df27c1291e86c699ddc25bc816.png" alt="三月七_无语" width="75" >

**Comment by chihung3927** [comment]
根据上述的理论,可以得出总结 米哈游快不行了 ,经典的滑坡理论

**Comment by excellen114514** [comment]
其实纯纯就是黑子没话找话了 咱只能说原处的环境太杂 见仁见智属于是

**Comment by lllliufeng** [comment]
世界上最长的路——米哈游的下坡路（

**Comment by KFGAS** [comment]
我一个黑子都不想看了，就觉得恶心

**Comment by kk5160** [comment]
那个女员工言论问题挺大的，但有的黑子却是趁机滑坡到各种其他角色在那恶心人，自我狂欢。

**Comment by 369852147abc** [comment]
大家只需要知道一句话“米哈游玩家自扩圈从过去以后成分非常复杂，他们不是一个整体每次节奏发声的人也不一样。”把节奏统一丢给黑子，友商是非常不理智的行为。

**Comment by 369852147abc** [comment]
大伟哥都说他们制作组在努力的分辨出玩家的诉求

**Comment by xiaoyujun1013** [comment]
皮套论就是高中的唯物论都没学明白的乱带<img src="https://bbs-static.miyoushe.com/static/2023/05/31/362c6ca877f65d820263eceb0517049a_3511022826083000682.png" alt="希儿 怒" width="75" >，而高管的言论就不好评价了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by sfncdsvncosdnovnfosdsvn** [comment]
就是21世纪莫须有，欲加之罪何患无辞

**Comment by RaphaelHider** [comment]
我不管，流萤和星就是宋媛媛和陈婷婷的皮套，我游戏里外狠狠磕，我磕磕磕磕磕磕磕磕磕磕磕磕磕。
![微信截图_20240819152455](https://github.com/user-attachments/assets/c0c76f59-e05d-4b27-a08f-d0b4c75cd2c8)
咳咳，当然了事实上肯定是先有角色再选的CV。
皮套节奏这个事，我认为世界上绝大多数文艺作品中的人物不是完全架空的，或多或少映射着作者或作者周围人的过去、现状或是期望，这是很正常的。
皮套节奏反智乐子程度个人评价0.3赛博龙脉，黑子妄想通过这个就让大量玩家破防属实是闹麻了。

**Comment by Pheathon** [comment]
星穹铁道的文案储备罢了

**Comment by yivitte** [comment]
不讨论，喜欢玩的不会走不爱玩的不会留，不过是找个理由从这个群体转到那个群体罢了

**Comment by jianjue1** [comment]
1：滚滚滚滚滚，爬爬爬爬爬
2：打过来了？我先跑（然后形成踩踏事故
<img src="https://bbs-static.miyoushe.com/static/2024/07/31/297dfcb691d834ce4960541412484f43_1488965655979817584.png" alt="椒丘 撒椒" width="75" >

**Comment by HyperMuteki0347** [comment]
赛博龙脉的含金量啧啧

**Comment by fyhwsy** [comment]
奇葩的很<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >
反正就是nt，不知道再吵什么

**Comment by warning-Kirov-reporting** [comment]
看那张集合图愣是没看懂，这节奏我直接免疫了<img src="https://bbs-static.miyoushe.com/static/2024/03/21/cfc12c6552ad36ddf88bd651e90da441_8544373733813917056.png" alt="丽都漫步-苍角" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/03/21/cfc12c6552ad36ddf88bd651e90da441_8544373733813917056.png" alt="丽都漫步-苍角" width="75" >

**Comment by Lyou777** [comment]
这些节奏只能影响圈外人的印象，米哈游实际怎么样，原神实际怎么样只有玩家清楚，只希望制作组不要被影响，继续产出优质的内容就好了，好游戏自然会有人买单

**Comment by Kamisato100** [comment]
同名是皮套人，生日是皮套人，文案是皮套人，CV是皮套人，美工是皮套人，动捕是皮套人，宣发是皮套人，一个角色这么多皮套原型啊，人均加拉赫是吧<img src="https://img-static.mihoyo.com/communityweb/upload/460474087917ad69a4ff975a3132a0df.png" alt="妮可 哈哈" width="75" >

带这些节奏和真信了这些节奏的，评价为大脑完全不发育，小脑发育不完全。左脑缺失运行库，右脑缺失指令集。<img src="https://bbs-static.miyoushe.com/static/2023/11/16/44d02375dbff4fde5ea3eadc82730131_8648553945620661617.png" alt="厨师邦布-疲惫" width="75" >

**Comment by HFOxin** [comment]
我从未见过这样的节奏，只能说当代米黑后继无人了。事件已经荒唐到开虚拟角色的黄谣，以及攻击现实工作人员了，这是最下流、无耻之徒能够说出来的最下贱、肮脏的话

**Reply by Robin-txdy** [reply]
点了

**Comment by 3165341164** [comment]
我真的被恶心吐了，并且我的同学好对此深信不疑，他明明知道我是托帕厨还要把托帕的那个所谓皮中人发给我，真的被恶心坏了，别人说什么就是什么呗，我还说我是砂金的皮中人呢<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/40f465ef5312e4ba0b19c596b9999cef_8760502457253372865.png" alt="三月七 无语" width="75" >

**Comment by Fluorine-z** [comment]
我只能给到0.2龙脉

**Comment by cuijiaqi0406** [comment]
氢原子核是所有物质的皮套.jpg<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by manyshoes** [comment]
太公式了，都没成梗，感觉还不如分不清虚幻和现实，这个起码有人跟风

**Comment by XPPPPO** [comment]
只能评价为智力缺陷型节奏，没证据链没线索没影响没原因，全tm是猜的。看一眼都多余。

**Comment by fzdqz** [comment]
一群挠馋带的纸张节奏，按它们的思路我甚至能推测出怀炎将军是散兵的皮套

**Comment by For-elysia** [comment]
虽然我没刷到过相关视频，但他是不是可以写进崩铁哈哈

**Comment by nabeilishi** [comment]
没活硬整，好笑程度0.25龙脉

**Comment by FAIR-MOONLIGHT** [comment]
谁信了，我笑他一坤年

**Comment by aino2333** [comment]
这有啥好讨论的，文艺创作中原型的存在是必不可少的。按照他们的逻辑任何文学形象都是皮套了
